_config.yml

```diff
@@ -75,6 +76,7 @@ social:
   links:
     github: bitcoindesign
     slack: bitcoindesign
+    discord: bitcoindesign
```

Comment:
Where is this used? Is this alt text?

Reply:
I have no idea. From what I can tell, nowhere. I just made a Discord one because there was a Slack one, but it might be old unused stuff.

Reply:
Git blame shows @GBKS added it in the early days, but from what I can see, it's not used anywhere on the site. Removed it from the config, rebuilt the site, ran rake test -- everything appears to work fine.

Not critical to remove it, but if it's not doing anything, doesn't need to be there.

Reply:
Thanks @GBKS, I reverted it back to the way it was (with the inclusion of a `discord` item in the social data).